engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionCallbacks.java

```diff
@@ -145,7 +145,7 @@ private String typeOf(Object value) {
       resultType = Constants.UNRESOLVED_SYMBOL;
     } else {
       var typeOfNode = TypeOfNode.getUncached();
-      Object typeResult = typeOfNode.execute(value);
+      Object typeResult = value == null ? null : typeOfNode.execute(value);
```

Comment:
Seems unrelated :thinking:

Reply:
Yup it is. However it is already already integrated as #8044, so there is no need to remove it.